src/test/java/easy/mind/com/api/service/BeckDepressionInventoryTests.java

```diff
@@ -57,62 +58,65 @@ public void createBeckDepressionInventory() {
     }
 
     @Test
-    public void createBeckDepressionInventoryWithDefunctUserId() {
+    public void create_withNotExistedUserId() {
         // given
         BeckDepressionInventory expectedInventory = createBeckDepressionInventoryForTesting();
         when(userService.readById(3)).thenThrow(new EntityNotFoundException("User with id 3 not found"));
         // when
         EntityNotFoundException thrown = assertThrows(
                 EntityNotFoundException.class,
-                () -> service.create(BeckDepressionToDTO.convert(expectedInventory),3),
-                "Expected readById() to throw, but it didn't"
+                () -> service.create(BeckDepressionToDTO.convert(expectedInventory), 3),
+                "Expected getByUserId() to throw, but it didn't"
         );
         // then
-        assertTrue(thrown.getMessage().contains("User with id"));
+        assertTrue(thrown.getMessage().contains("User with id 3 not found"));
 
         Mockito.verify(userService).readById(3L);
         Mockito.verify(repository, never()).save(any());
 
     }
 
     @Test
-    public void getByUserId(){
+    public void get_successfulByUserId() {
 
         // given
         User user = createUserForTesting();
 
         BeckDepressionInventory expectedInventory = createBeckDepressionInventoryForTesting();
-        List<BeckDepressionInventory> usersListWithInventories = new ArrayList<>();
-        usersListWithInventories.add(expectedInventory);
-
+        List<BeckDepressionInventory> usersListWithInventories = List.of(expectedInventory);
+        List<BeckDepressionInventoryDTO> usersListWithInventoriesDTO =
```

Comment:
It's only one entry, simplify it: `List.of(BeckDepressionToDTO.convert(expectedInventory))`